Cargo.toml

```diff
@@ -77,3 +77,7 @@ substrate-build-script-utils = '2.0.0'
 [features]
 default = []
 runtime-benchmarks = ['node-template-runtime/runtime-benchmarks']
+
+[patch.crates-io]
+prost = { git = "https://github.com/adoriasoft/prost", rev = "389010a0a79ef2a5805f510a76108e531c1fb621" }
```

Comment:
Replace " with ' as in rest of cargo tomls

Reply:
fixed